cmux-tui/bindings/tests/test_verify_crates_ownership.py

```
@@ -0,0 +1,106 @@
+                self.owner_login,
+            )
+
+    def test_rejects_a_wrong_repository(self) -> None:
+        def response(request: object, **kwargs: object) -> io.BytesIO:
+            payload = json.loads(self.registry_response(request, **kwargs).read())
+            if not str(getattr(request, "full_url", "")).endswith("/owners"):
+                payload["crate"]["repository"] = "https://github.com/attacker/cmux"
+            return self.response(payload)
+
+        with mock.patch.object(ownership, "urlopen", side_effect=response), \
+            self.assertRaisesRegex(ownership.OwnershipError, "repository"):
```

Ruff (0.16.0)

[warning] Use `pytest.raises` instead of unittest-style `assertRaisesRegex`

Replace `assertRaisesRegex` with `pytest.raises`

(PT027)